sky/spot/spot_state.py

```diff
@@ -22,6 +22,9 @@
 _CONN = sqlite3.connect(str(_DB_PATH))
 _CURSOR = _CONN.cursor()
 
+# `spot` table contains all the finest-grained jobs, including all the
+# sub jobs of a spot job. All sub jobs will have the same `new_job_id`.
+# The `job_name` is the name of the sub job.
```

Comment:
- Update terms in comments?
- `new_job_id` -> something better? `encapsulating_job_id`? `spot_job_id`?

Reply:
- Updated
- Changed to `spot_job_id`